pom.xml

```diff
@@ -21,7 +21,7 @@
     <groupId>org.robotframework</groupId>
     <artifactId>javafxlibrary</artifactId>
     <packaging>jar</packaging>
-    <version>0.7.1-SNAPSHOT</version>
+    <version>0.7.1</version>
```

Comment:
Development branch should be SNAPSHOT, this could be 0.7.1-SNAPSHOT until changed to new version.

Reply:
Fixed